README.md

```diff
@@ -7,7 +7,12 @@ Package license: GPL3
 
 Feedstock license: BSD 3-Clause
 
-Summary: The silx project aims at providing a collection of Python packages to support the development of data assessment, reduction and analysis applications at synchrotron radiation facilities.
+Summary: Dioptas is a Python-based program for on-the-fly data processing 
```

Comment:
This is auto generate when the recipe is re-rendered, so only your change in the `meta.yaml` is needed.